openmrs-19/src/main/java/org/motechproject/openmrs19/service/OpenMRSRelationshipService.java

```diff
@@ -0,0 +1,21 @@
+package org.motechproject.openmrs19.service;
+
+import org.motechproject.openmrs19.domain.Relationship;
+
+/**
+ * Interface for handling relationships on the OpenMRS server.
+ */
+public interface OpenMRSRelationshipService {
+
+    /**
+     * Returns relationship of the type with the given {@code typeUuid} for the person with the given
+     * {@code personUuid}.
+     *
+     * @param configName  the name of the configuration to be used while performing this action
+     * @param typeUuid  the UUID of the type
+     * @param personUuid  the UUID of the person
+     * @return the matching relationship
+     */
+    Relationship getByTypeUuidAndPersonUuid(String configName, String typeUuid, String personUuid);
```

Comment:
Methods without config name as parameters have been removed. With @kmadej changes we're losing backward compatibility, so those methods become redundant. I couldn't think of any other use case for them.